content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -111,18 +112,23 @@ case, this is the `metaplex-foundation/mpl-token-metadata`.
 import { mplTokenMetadata } from "@metaplex-foundation/mpl-token-metadata";
 import { keypairIdentity } from "@metaplex-foundation/umi";
 import { createUmi } from "@metaplex-foundation/umi-bundle-defaults";
-import { getKeypairFromFile } from "@solana-developers/helpers";
-import { promises as fs } from "fs";
-import { clusterApiUrl } from "@solana/web3.js";
+import fs from "fs";
+import { clusterApiUrl, Keypair } from "@solana/web3.js";
 
 const umi = createUmi(clusterApiUrl("devnet"));
 
 // load keypair from local file system
-// See https://github.com/solana-developers/helpers?tab=readme-ov-file#get-a-keypair-from-a-keypair-file
-const localKeypair = await getKeypairFromFile();
+const walletFile = JSON.parse(
+  fs.readFileSync(
+    path.resolve(__dirname, "/home/user/.config/solana/id.json"),
+    "utf-8",
+  ),
+);
+
+const user = Keypair.fromSecretKey(new Uint8Array(walletFile));
```

Comment:
Why not just use `await getKeypairFromFile();`? This PR makes the process of loading a keypair more complicated.